code/modules/reagents/chemistry/machinery/chem_heater.dm

```diff
@@ -88,7 +88,10 @@
 
 /obj/machinery/chem_heater/screwdriver_act(mob/user, obj/item/I)
 	. = TRUE
-	default_deconstruction_screwdriver(user, "mixer0b", "mixer0b", I)
+	if(!beaker)
+		default_deconstruction_screwdriver(user, "mixer0b", "mixer0b", I)
+	else
+		default_deconstruction_screwdriver(user, "mixer0b", "mixer1b", I)
```

Comment:
```suggestion
	default_deconstruction_screwdriver(user, "mixer0b", "mixer[beaker ? "1" : "0"]b", I)
```
This will add add a `1` if `beaker` exists. Else it'll be a `0`